packages/doom_emacs.rb

```diff
@@ -0,0 +1,91 @@
+require 'package'
+
+class Doom_emacs < Package
+  description 'An Emacs framework for the stubborn martian hacker'
+  homepage 'https://github.com/hlissner/doom-emacs'
+  version '27.0.91'
+  source_url 'http://git.savannah.gnu.org/cgit/emacs.git/snapshot/emacs-27.0.91.tar.gz'
+  source_sha256 'ac7b71927dfdfa3b9c034da82e827ee28e7467d58953dafed69e9ce9d700cebd'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'giflib'
+  depends_on 'libjpeg'
+  depends_on 'libpng'
+  depends_on 'libtiff'
+  depends_on 'cairo'
+  depends_on 'libxaw'
+  depends_on 'libxpm'
+  depends_on 'wxwidgets'
+  depends_on 'imagemagick6'
+  depends_on 'sommelier'
+
+  def self.build
+    system './autogen.sh'
+    system './configure',
+            "--prefix=#{CREW_PREFIX}",
+            "--localstatedir=#{CREW_PREFIX}/share",
+            '--without-selinux',
+            '--with-x=yes',
+            '--with-x-toolkit=gtk',
+            '--with-gif=yes',
+            '--with-jpeg=yes',
+            '--with-png=yes',
+            '--with-rsvg=yes',
+            '--with-xaw=yes',
+            '--with-xpm=yes',
+            '--with-tiff=yes',
+            '--without-makeinfo',
+            '--with-pdumper',
+            '--with-gnutls',
+            '--with-cairo'
+    system 'make'
+  end
+
+  def self.install
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system 'install', '-Dm755', 'src/emacs', "#{CREW_DEST_PREFIX}/bin/gemacs"
+    system 'install', '-Dm755', 'src/emacs-27.0.91.1', "#{CREW_DEST_PREFIX}/bin/gemacs-27.0.91"
+    if Dir.exist?("#{HOME}/.emacs.d")
+      puts "~/.emacs.d already exists. Delete it ? (y/n) ".orange
+      while answer = STDIN.gets.chomp
+        case answer
+          when 'y'
+            puts "Removing ~/.emacs.d...".orange
+            FileUtils.remove_dir("#{HOME}/.emacs.d",true) 
+            break
+          when 'n'
+            puts "Doom_emacs not installed. Please remove ~/.emacs.d".lightred
+            abort
+            break
+          else
+            puts "  Please select from one of the options."
+        end
+      end
+    end
+    system "git clone --depth 1 https://github.com/hlissner/doom-emacs #{HOME}/.emacs.d"
```

Comment:
Should we checkout a tagged release here?  Otherwise, this is on the master (or default) branch which may change over time and cause potential incompatibilities.

Reply:
`system "git clone --depth 1 https://github.com/hlissner/doom-emacs #{CREW_DEST_HOME}/.emacs.d"`?